### DISCUSSION TITLE
MSBUILD : error MSB1003: Specify a project or solution file.

### DISCUSSION THREAD

**Discussion by fataIlty** [winstxnhdw/lc-hax#412] [Q&A] [answered]
### What happened?

Hello, i downloaded all requirements and then i opened requirements.bat and this error has appeared :
MSBUILD : error MSB1003: Specify a project or solution file.

### Current Commit Hash

I cant find HEAD

### Injector

- [X] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have confirmed that my anti-virus is not blocking any of the relevant programs
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
Just watch the installation video: https://github.com/winstxnhdw/lc-hax/discussions/389

**Comment by fataIlty** [comment]
I did everything in the video and I still get the same error

**Comment by winstxnhdw** [comment]
Did you clone the repository? And did you read this very big note:

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/a436ce45-c6a5-4b71-9669-0961377bb0d1)

Is your `submodules/SharpMonoInjectorCore` folder empty? If it's not empty, check your anti-virus. But it is more likely that it is empty. This means your recursive clone failed. It can only fail if your Git install is broken. Your issue might be similar to the guy in #293.

**Comment by fataIlty** [comment]
Yeah its empty, i was tryed to open launch.bat and i got error : Can't find "./submodules/SharpMonoInjectorCore/dist/SharpMonoInjector.exe"
Yes,i cloned repository and read note

**Comment by winstxnhdw** [comment]
> I cant find HEAD

If you can't find the HEAD, it means you never cloned the repository.

**Reply by fataIlty** [reply]
I cloned the repository a long time ago.

**Reply by winstxnhdw** [reply]
Then you should be able to find HEAD. If you are that certain, then delete the clone and re-clone it.

**Reply by fataIlty** [reply]
uh, ig im stupid bc idk how to delete clone

**Reply by winstxnhdw** [reply, marked as answer]
A clone is just a folder that is created wherever you clone it. How about you try following these instructions instead: https://github.com/winstxnhdw/lc-hax/discussions/294

**Reply by fataIlty** [reply]
Omg bro, it worked, I opened the requirements.bat and it all worked
